### PR TITLE
README.md: Describe game specific settings

### PR DESCRIPTION
**Pull request by kakurasan**
For #208

### CODE REVIEW

**Comment by Lucki** [comment]
The `[DEFAULT]` section needs a mention and the example file could get a bit extended with a few easy examples.

Also we should declutter these tables a bit. Please evaluate putting defaults and notes into [new lines](https://stackoverflow.com/questions/11700487/newline-in-markdown-table) or their own rows (this will probably make the table to small to be useful).
This will probably improve readability and allows to get rid of a bunch of brackets.

The default configuration file path could get a mention in the default paths section.

File: README.md [hunk 1]
```diff
@@ -253,7 +253,17 @@ Supported sections are `[ats]`, `[atsmp]`, `[ets2]` and `[ets2mp]`.
 
 Key-Value-Pair|Description
 ---|---
-`without-rich-presence = [boolean]`|Disable Discord Rich Presence for the game mode - overriding what is set in third party sections.
+`without-rich-presence = [boolean]`|Disable Discord Rich Presence for the game mode - overriding what is set in third party sections. [Default: `no`]
+`game-directory = [path]`|Choose a different directory for the game files [Default: `$XDG_DATA_HOME/truckersmp-cli/(Game name)/data`]
+`prefix-directory = [path]`|Choose a different directory for the prefix [Default: `$XDG_DATA_HOME/truckersmp-cli/(Game name)/prefix`]
+`game-options = [options]`|Specify ATS/ETS2 options Note: If specifying one option, use `--game-options=-option` format [Default: `-nointro -64bit`]
```
Comment (Lucki) [inline README.md:259]: I'm not entirely sure what you're trying to say here.

Should the config entry look like `game-options = --game-options=-option` or **has** this to be a parameter in the program call `truckersmp-cli --game-options=-option`?

```suggestion
`game-options = [options]`|Specify ATS/ETS2 options Note: If specifying only one option, use `--game-options=-option` format [Default: `-nointro -64bit`]
```

File: README.md [hunk 2]
```diff
@@ -253,7 +253,17 @@ Supported sections are `[ats]`, `[atsmp]`, `[ets2]` and `[ets2mp]`.
...
+`rendering-backend = {auto,dx11,gl}`|Choose a rendering backend [Default: `auto` (OpenGL is used when `rendering-backend = ` is not specified for the game in the configuration file)]
```
Comment (Lucki) [inline README.md:260]:
```suggestion
`rendering-backend = {auto,dx11,gl}`|Choose a rendering backend [Default: `auto` (OpenGL is used when this is not specified for the game in the configuration file)]
```

**Comment by kakurasan** [comment]
Some changes unrelated to game specific settings are not applied in *this* PR: Move to #273.

**Pull request merged**